README.md

```diff
@@ -60,24 +60,20 @@ Requests made with a missing or invalid token will get a `401 Unauthorized` resp
 ## Usage
 
 ```javascript
-import Shortcut from 'clubhouse-lib';
+import { ShortcutApi } from '@useshortcut/client';
```

Comment:
Not sure if you welcome nit-picky comments, but here it goes :-)

Feel free to disregard it.

This is a client. It is not an API. The API is a specification for _how_ the client talks to the server.

Suggestion 1: renaming this to `ShortcutClient`, or even shorter just `Shortcut`.

Suggestion 2: Consider making it the default export. Then first suggestion isn't even an issue, we can do `import iCanNameItWhateverIWant from '@useshortcut/client'`

Reply:
Of course, any comments are welcome!

Thanks for providing this feedback, that's exactly why I tagged you all. Naming isn't easy and naming things right is crucial before we release this.

> This is a client. It is not an API. The API is a specification for how the client talks to the server.
>
> Suggestion 1: renaming this to `ShortcutClient`, or even shorter just `Shortcut`.

I agree. I'll go with `ShortcutClient` to make it consistent with the expected type.

> Suggestion 2: Consider making it the default export. The first suggestion isn't even an issue, we can do `import iCanNameItWhateverIWant from '@useshortcut/client'`

I'm not sure how I feel about default export. I tend to avoid them, especially with TypeScript. We can still do: `import { ShortcutClient: iCanNameItWhateverIWant} from '@useshortcut/client'`, but I can see the point of avoiding the extra characters. 🤔

Reply:
Agree. Naming is one of the hardest things to get right!

As for default export, It probably comes down to preference.

One consideration, perhaps, might be that [the majority](https://madnight.github.io/githut/#/pull_requests/2021/2) of your end-users are likely to use vanilla javascript.

In the EcmaScript Modules world (where I live and breathe) it is the norm to have a default export. I don't have stats to back this up, just my own experience.